Formula/pinocchio.rb

```diff
@@ -0,0 +1,50 @@
+class Pinocchio < Formula
+  desc "Efficient and fast C++ library implementing Rigid Body Dynamics algorithms"
+  homepage "https://stack-of-tasks.github.io/pinocchio"
+  url "https://github.com/stack-of-tasks/pinocchio/releases/download/v2.6.10/pinocchio-2.6.10.tar.gz"
+  sha256 "4ded587192561d434d4158fdae56dd36b0c3a0db65ae4768df94428161bad522"
+  license "BSD-2-Clause"
+  head "https://github.com/stack-of-tasks/pinocchio.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "boost"
+  depends_on "eigen"
+  depends_on "eigenpy"
+  depends_on "hpp-fcl"
+  depends_on "python@3.10"
+  depends_on "urdfdom"
+
+  def python3
+    deps.map(&:to_formula)
+        .find { |f| f.name.match?(/^python@\d\.\d+$/) }
+        .opt_libexec/"bin/python"
+  end
```

Comment:
This is only ever going to return `Formula["python@3.10"].opt_libexec/"bin/python"` so you can just use that

Reply:
Thanks @SMillerDev for the quick feedback.